app/helpers/provider_helper.rb

```diff
@@ -13,10 +13,6 @@ def create_course_link(provider, **opts)
     govuk_link_to("Add a new course", new_provider_recruitment_cycle_course_path(provider_code: provider.provider_code, recruitment_cycle_year: provider.recruitment_cycle_year), class: "govuk-button govuk-!-margin-bottom-2", **opts)
   end
```

Comment:
As we are only using this twice in `app/views/courses/index.html.erb`, I think we should remove this helper method and just call this twice.  If we needed to use a create course link would we remember that this helper even exists if its only used twice?